src/components/index.js

```diff
@@ -0,0 +1,9 @@
+import Navbar from "./navbar/Navbar";
+import Slider from './slider/Slider';
+import CatagoryCard from "./catagories/CatagoryCard";
+import CountDown from "./countDown/CountDown";
+import Footer from "./footer/Footer";
+import Brands from './brands/Brands';
+import ProductCard from "./productCard/ProductCard";
+import Filter from "./filters/Filter";
+export{Navbar,Slider,CatagoryCard,CountDown,Footer,Brands,ProductCard,Filter}
```

Comment:
Named exports allow us to export directly as shown below. So we don't have to import it first and then export it. You can try this if you wish to.

`export { Navbar } from "./navbar/Navbar";`

Reply:
Done!! all the suggested changes are made